docs/index.html

```diff
@@ -74,24 +74,24 @@ <h2 class="mt-4">Screenshots</h2>
                 </div>
                 <div data-u="slides" style="cursor:default;position:relative;top:0px;left:0px;width:980px;height:550px;overflow:hidden;">
                     <div>
-                        <img data-u="image" src="https://raw.githubusercontent.com/RITlug/TigerOS/master/screenshots/apps.png" />
-                        <img data-u="thumb" src="https://raw.githubusercontent.com/RITlug/TigerOS/master/screenshots/apps.png" />
+                        <img data-u="image" src="https://raw.githubusercontent.com/RITlug/TigerOS/master/screenshots/TigerOS_Orange_Background.png" />
+                        <img data-u="thumb" src="https://raw.githubusercontent.com/RITlug/TigerOS/master/screenshots/TigerOS_Orange_Background.png" />
```

Comment:
As a convention, I normally keep my file names all lowercase. I don't have any other reason better than I think it looks nicer in a URL string and is easier to type. Take it as you may. :slightly_smiling_face: